test/fixtures/mock-sdk/lib/toolkit/loader.js

```diff
@@ -759,14 +769,14 @@ const Loader = iced(function Loader(options) {
     load: { enumerable: false, value: options.load || load },
     // Main (entry point) module, it can be set only once, since loader
     // instance can have only one main module.
-    main: new function() {
+    main: function() {
```

Comment:
oh if it's a fixture and the tests pass then meh. It was probably just a typo, better to fix it.